Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```
@@ -31,6 +31,26 @@ configuration:
   name: private_key
   required: false
   type: 14
+- additionalinfo: Type of authentication - could be Authorization Code Flow (recommended) or Device Code Flow.
```

```suggestion
- additionalinfo: Type of authentication - either Authorization Code (recommended) or Client Credentials.
```
why is auth code the recommended one?